src/Opg/Lpa/Pdf/Service/Forms/Lp3.php

```diff
@@ -81,95 +62,43 @@ public function generate()
      *
      * @param NotifiedPerson $peopleToNotify
      */
-    protected function generateSingleNotificationPdf()
+    protected function generateSingleNotificationPdf(NotifiedPerson $peopleToNotify)

```

Comment:
Change to $personToNotify please
Also can the function be changed from generateSingleNotificationPdf to generateNotificationPdf as this logic will execute for many people to notify as well as a single party